packages/firestore/src/platform_node/node_platform.ts

```diff
@@ -15,72 +15,71 @@
-  }
+/** Loads the GRPC stack */
+export function loadConnection(
+  databaseInfo: DatabaseInfo
+): Promise<Connection> {
+  const protos = loadProtos();
+  return Promise.resolve(new GrpcConnection(protos, databaseInfo));
+}
 
-  loadConnection(databaseInfo: DatabaseInfo): Promise<Connection> {
-    const protos = loadProtos();
-    return Promise.resolve(new GrpcConnection(protos, databaseInfo));
-  }
+/** Return the Platform-specific connectivity monitor. */
+export function newConnectivityMonitor(): ConnectivityMonitor {
+  return new NoopConnectivityMonitor();
+}
 
-  newConnectivityMonitor(): ConnectivityMonitor {
-    return new NoopConnectivityMonitor();
-  }
+/** Return the Platform-specific serializer monitor. */
+export function newSerializer(databaseId: DatabaseId): JsonProtoSerializer {
+  return new JsonProtoSerializer(databaseId, { useProto3Json: false });
+}
 
-  newSerializer(partitionId: DatabaseId): JsonProtoSerializer {
-    return new JsonProtoSerializer(partitionId, { useProto3Json: false });
+/** Converts a Base64 encoded string to a binary string. */
+export function decodeBase64(encoded: string): string {
+  // Node actually doesn't validate base64 strings.
+  // A quick sanity check that is not a fool-proof validation
+  if (/[^-A-Za-z0-9+/=]/.test(encoded)) {
+    throw new FirestoreError(
+      Code.INVALID_ARGUMENT,
+      'Not a valid Base64 string: ' + encoded
+    );
   }
+  return new Buffer(encoded, 'base64').toString('binary');
+}
 
-  formatJSON(value: unknown): string {
-    // util.inspect() results in much more readable output than JSON.stringify()
-    return inspect(value, { depth: 100 });
-  }
+/** Converts a binary string to a Base64 encoded string. */
+export function encodeBase64(raw: string): string {
+  return new Buffer(raw, 'binary').toString('base64');
+}
 
-  atob(encoded: string): string {
-    // Node actually doesn't validate base64 strings.
-    // A quick sanity check that is not a fool-proof validation
-    if (/[^-A-Za-z0-9+/=]/.test(encoded)) {
-      throw new FirestoreError(
-        Code.INVALID_ARGUMENT,
-        'Not a valid Base64 string: ' + encoded
-      );
-    }
-    return new Buffer(encoded, 'base64').toString('binary');
+/** The Platform's 'window' implementation or null if not available. */
+export function getWindow(): Window | null {
+  if (process.env.USE_MOCK_PERSISTENCE === 'YES') {
+    // eslint-disable-next-line no-restricted-globals
+    return window;
   }
 
-  btoa(raw: string): string {
-    return new Buffer(raw, 'binary').toString('base64');
-  }
+  return null;
+}
 
-  randomBytes(nBytes: number): Uint8Array {
-    debugAssert(nBytes >= 0, `Expecting non-negative nBytes, got: ${nBytes}`);
+/** The Platform's 'document' implementation or null if not available. */
+export function getDocument(): Document | null {
+  return null;
+}
 
-    return randomBytes(nBytes);
-  }
+/** True if and only if the Base64 conversion functions are available. */
```

Comment:
This seems related to the other base64 methods--maybe group these together?

Reply:
They are now group in the "base64" module.